src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx

```diff
@@ -0,0 +1,297 @@
+        justifyContent: 'center',
+      },
+      '.confidence': {
+        backgroundColor: Colors.Background200Dark,
+      },
+      '.comparison-sign': {
+        backgroundColor: Colors.Background200Dark,
+      },
+      '.delta': {
+        backgroundColor: Colors.Background200Dark,
+      },
+      '.expand-button-container': {
+        justifyContent: 'right',
+      },
+      '.new-value': {
+        backgroundColor: Colors.Background200Dark,
+      },
+      '.subtests': {
+        backgroundColor: Colors.Background200Dark,
+        borderRadius: '4px 0 0 4px',
+        paddingLeft: Spacing.xLarge,
+        justifyContent: 'left',
+      },
+      '.subtests-container': {
+        alignItems: 'flex-end',
+        backgroundColor: Colors.Background200Dark,
+        display: 'flex',
+      },
+      '.status': {
+        backgroundColor: Colors.Background200Dark,
+        justifyContent: 'center',
+      },
+      '.total-runs': {
+        backgroundColor: Colors.Background200Dark,
+      },
+      '.row-buttons': {
+        backgroundColor: Colors.Background200Dark,
+        borderRadius: '0px 4px 4px 0px',
+        display: 'flex',
+        justifyContent: 'flex-end',
+        $nest: {
+          '.download': {
+            cursor: 'not-allowed',
+          },
+        },
+      },
+      '.expand-button': {
+        backgroundColor: Colors.Background100Dark,
+      },
+    },
+  }),
+  typography: style({
+    ...typography,
+  }),
+};
+
+const stylesCard = ExpandableRowStyles();
+
+function determineStatus(improvement: boolean, regression: boolean) {
+  if (improvement) return 'Improvement';
+  if (regression) return 'Regression';
+  return '-';
+}
+
+function determineSign(baseMedianValue: number, newMedianValue: number) {
+  if (baseMedianValue > newMedianValue) return '>';
+  if (baseMedianValue < newMedianValue) return '<';
+  return '';
+}
+
+function SubtestsRevisionRow(props: RevisionRowProps) {
+  const { result } = props;
+  const {
+    test,
+    base_median_value: baseMedianValue,
+    base_measurement_unit: baseUnit,
+    new_median_value: newMedianValue,
+    new_measurement_unit: newUnit,
+    is_improvement: improvement,
+    is_regression: regression,
+    delta_percentage: deltaPercent,
+    confidence_text: confidenceText,
+    base_runs: baseRuns,
+    new_runs: newRuns,
+    graphs_link: graphLink,
+  } = result;
+
+  const [expanded, setExpanded] = useState(false);
+
+  const toggleIsExpanded = () => {
+    setExpanded(!expanded);
+  };
+
+  const themeMode = useAppSelector((state) => state.theme.mode);
+
+  const styles = themeMode === 'light' ? stylesLight : stylesDark;
+
+  return (
+    <>
+      <div
+        className={`revisionRow ${styles.revisionRow} ${styles.typography}`}
+        role='row'
+      >
+        <div className='subtests cell' role='cell'>
+          <div className='subtests-container'>{test}</div>
+        </div>
+        <div className='base-value cell' role='cell'>
+          {' '}
```

Comment:
I added this suggestion in the clean-up [subtask](https://mozilla-hub.atlassian.net/browse/PCF-488) and it will be addressed later on for both components.